draft-ietf-quic-http.md

```diff
@@ -1335,8 +1343,8 @@ The following settings are defined in HTTP/3:
   : The default value is unlimited.  See {{header-formatting}} for usage.
 
   SETTINGS_NUM_PLACEHOLDERS (0x9):
-  : The default value is 0.  However, this value SHOULD be set to a non-zero
-    value by servers.  See {{placeholders}} for usage.
+  : The default value is 0.  However, servers SHOULD set this value to 32 or
```

Comment:
Wow, 32 is a large number.  If we keep exclusive prioritization, then Chrome will probably continue to use that, and Firefox doesn't use this many, so why such a high limit?

Reply:
Just because we use 6 now, doesn't mean that we'll keep that scheme.  We're currently looking at ways to improve prioritization and placeholders are seen as being very useful in doing that.  It's very early yet, but it might be that more are found to be useful.

And 32 isn't a big number.  You need quite a few more zeros to qualify, I'd think.

Reply:
FWIW, I do not care much, because the amount of state required by each placeholder is much less than that required by each HTTP stream (which includes send buffers). We recommend servers supporting at least 100 concurrent HTTP streams, so 32 placeholders can be a SHOULD.

That said, I think I might prefer choosing a number that is a divisor of 100 (e.g., 20, 25).

Reply:
Powers of two seem more natural in a protocol, but I am similarly uncaring about the specific number.

Reply:
I also suggested 32 because the "power of two" feeling. Otoh, I was thinking about Patrick Meenan's setup, which requires 4 placeholders per priority level in the current scheme, allowing for 8 priority levels if the minimum number of placeholders is 32. 8 is more than e.g., the 5 chrome is using at the moment, but seems like a good number (see also SPDY priority levels).

I just want to prevent us setting this to a too low a number (e.g., 10), since that would counteract the purpose of setting a lower bound imo.

Reply:
While I think it makes more sense to define a number that is defined in relation to the number of concurrent requests we suggest to support, I do not think that the current value of 100 is correct (see #2787).

Considering that, I am fine with seeing this PR merged with 32 being the advice, and revisiting the advisory value if necessary.